client/asset/btc/btc.go

```diff
@@ -502,8 +530,17 @@ func newWallet(requester RawRequester, cfg *BTCCloneCFG, btcCfg *dexbtc.Config)
 		useLegacyBalance:    cfg.LegacyBalance,
 		segwit:              cfg.Segwit,
 		legacyRawFeeLimit:   cfg.LegacyRawFeeLimit,
+		signNonSegwit:       nonSegwitSigner,
+		estimateFee:         cfg.FeeEstimator,
+		decodeAddr:          addrDecoder,
 		walletInfo:          cfg.WalletInfo,
-	}, nil
+	}
+
+	if w.estimateFee == nil {
+		w.estimateFee = w.feeRate
```

Comment:
The first time we call this, maybe even here in construction, if response is "method unknown" (or whatever the code is), I think it should be a hard error.  I say this because I was trying to run the BCH live_test.go and it actually passed the test with the BTC bitcoind except for an error about method unknown for the fee estimate.  Also makes me wonder if we should inspect some other RPC result to be sure we're even connecting to the correct asset.